Media/Thumbnail/Strategies/VideoToImageManager.php

```diff
@@ -17,18 +15,18 @@ class VideoToImageManager implements ThumbnailInterface
 
     protected $tmpDir;
     protected $mediaDirectory;
-    protected $ffmpegMovieFactory;
+    protected $ffmpegMovieFrame;
 
     /**
-     * @param string                               $tmpDir
-     * @param string                               $mediaDirectory;
-     * @param FFmpegMovieFactoryInterface $ffmpegMovieFactory;
+     * @param string                    $tmpDir
+     * @param string                    $mediaDirectory;
+     * @param FFmpegMovieFrameInterface $ffmpegMovieFrame;
      */
-    public function __construct($tmpDir, $mediaDirectory, FFmpegMovieFactoryInterface $ffmpegMovieFactory)
+    public function __construct($tmpDir, $mediaDirectory, FFmpegMovieFrameInterface $ffmpegMovieFrame)
     {
         $this->tmpDir = $tmpDir;
         $this->mediaDirectory = $mediaDirectory;
-        $this->ffmpegMovieFactory = $ffmpegMovieFactory;
+        $this->ffmpegMovieFrame = $ffmpegMovieFrame;
```

Comment:
With the previous remark on the interface name, here you should use something like ``videoManager`` instead of ffmpegMovieFrame